### DISCUSSION TITLE
Is this (or something similar) available without gnome?

### DISCUSSION THREAD

**Discussion by kovasap** [Schneegans/Burn-My-Windows#30] [Q&A]
I like using the QTile tiling window manager with compton for transparency.  Has anyone gotten this extension to work without Gnome or know of a similar extension that would work?

**Comment by Schneegans** [comment]
Something like this is necessarily quite deeply coupled with the window manager / desktop environment. I am not aware of any project which can do this in a cross-desktop-environmental matter. [Wayfire](https://wayfire.org/2019/01/13/Introduction-to-Wayfire.html) brings a lot of the old compiz stuff.

**Reply by kovasap** [reply]
Got it, thanks for the link!

**Comment by dinkopehar** [comment]
Does it work properly with XFCE ?

**Reply by Schneegans** [reply]
You can only use Burn-My-Windows on GNOME Shell 3.36, 3.38, 40, and 41. Currently, the T-Rex-Attack Shader and the Matrix-Shader are only supported on 40 and 41.
